README.rst

```diff
@@ -125,6 +125,31 @@ To download, setup and perfom tests, run the following commands on Mac / Linux:
  cd mongomock
  tox
 
+Alternatively, docker-compose can be used to simplify dependency management for local development:
+
+.. code-block:: bash
+
+ git clone git@github.com:mongomock/mongomock.git
+ cd mongomock
+ docker-compose build
+ docker-compose run --rm mongomock
+
+If you need/want tox to recreate its environments, you can override the container command by running:
+
+.. code-block:: bash
+
+ docker-compose run --rm mongomock tox -r
+
+Similarly, if you'd like to run tox against a specific environment in the container:
+
+.. code-block:: bash
+
+ docker-compose run --rm mongomock tox -e py35-pymongo-pyexecjs
```

Comment:
Please use the latest and more complete version as documentation (py38-pymongo-pyexecjs)